apps/src/templates/VerticalImageResourceCard.jsx

```diff
@@ -114,6 +114,8 @@ class VerticalImageResourceCard extends Component {
       require('@cdo/static/resource_cards/codeorgteacher.png'),
```

Comment:
nit: not sure what our linter says, but this is a bit hard to read with no indentation..